src/Makefile.am

```diff
@@ -1 +1,131 @@
-SUBDIRS = common fairness bindings plugins cmd
+SUBDIRS = common bindings plugins
+
+AM_CXXFLAGS = \
+    $(WARNING_CXXFLAGS) \
+    $(CODE_COVERAGE_CXXFLAGS)
+
+AM_LDFLAGS = $(CODE_COVERAGE_LDFLAGS) $(SQLITE_LIBS)
+
+AM_CPPFLAGS = -I$(top_srcdir) $(FLUX_CORE_CFLAGS)
+
+noinst_LTLIBRARIES = libweighted_tree.la
+noinst_HEADERS = \
+	fairness/account/account.hpp \
```

Comment:
Appears to be a whitespace error here (at least in Github view) (tab vs spaces?)

Reply:
Oops! Thanks for pointing that out - looked like there were multiple whitespace errors so I force pushed a fix to convert tabs to spaces.

Reply:
I think in a `Makefile` you may want to make sure to use tabs? It may work out ok for a Makefile.am but generally tabs are required in some places in Makefiles, so it might be safer to stick with tabs.